Explain whitespace trimming using different words (#138)
datacarpentry/openrefine-socialsci: _episodes/02-working-with-openrefine.md

## Trim Leading and Trailing Whitespace

Sometimes spaces (or tabs, or newline characters) will be present at the beginning
or end of a text cell.
They may have been in the dataset that was imported, or appear when you perform
operations on the data, such as splitting text.
While we as humans cannot always see or notice these (especially if they are at
the end of a word), a computer always sees them.

> We often see these spaces as unwanted variations and therefore remove them.

Reviewer: ```suggestion
These spaces are often unwanted variations that should to be removed.
```
